.prettierrc.js

```diff
@@ -0,0 +1,13 @@
+// This config attempts to match google-style code.
+
+module.exports = {
+  // Prefer single quotes, but minimize escaping.
+  singleQuote: true,
+  // Some properties must be quoted to preserve closure compiler behavior.
+  // Don't ever change whether properties are quoted.
+  quoteProps: 'preserve',
```

Comment:
Do we need this in samples?

Reply:
No, I don't think so. But keeping it as this value means prettier won't touch the quotes and I didn't want to do that in this PR either. I'll also add it to the future work section.